### PR TITLE
Added Navbar in Feedback Page

### LINKED ISSUE #469: Feature_Request:  Added missing Navbar on the feedback page
**Is your feature request related to a problem? Please describe.**

the Navbar was missing in the feedback section  and unable to navigate back so , itis important to add Navbar to feedback page also

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Demo Video/Screenshot (Mandatory )**
before screenshot
Screenshot (1692)

after updating

Screenshot (1693)

***Are you contributing under any open-source program ?***
yes girlscript summer of code (Gssoc'24)

### PR DESCRIPTION
**Pull request by SadafKausar2025**
# Title and Issue number 

Title :Added missing Navbar on the feedback page 

Issue No. :#469

Code Stack : 

Close #469

the Navbar was missing in the feedback section  and unable to navigate back so , itis important to add Navbar to feedback page also

**Describe the solution you'd like**

**Describe alternatives you've considered**

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have gone through the  `contributing.md` file before contributing

**Additional context (Mandatory )**

***Are you contributing under any Open-source programme?***
Gssoc24(girscript summer of code)

original page : without Navbar on Rating page

Screenshot (1692)

After updating

Screenshot (1693)

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
Hey @apu52 
I have created new PR with resolved Conflict.
Now its Ok

File: feedback.html [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
+          <li class="link home" data-aos="fade-down"><a href="./index.html#home" target="_blank">Home</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#AboutUs" target="_blank">AboutUs</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#Destinations" target="_blank">Destinations</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#trip" target="_blank">Pricing</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#ReviewGallery" target="_blank">Trip Gallery</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#testimonials" target="_blank">Testimonials</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.htmlfeedback.html" target="_blank">Rate Us?</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#contact" target="_blank">Contact Us</a></li>
 
         </ul>
```
Comment (apu52) [inline feedback.html:33]: @SadafKausar2025  

image

There is an unexpected error in the login page . Whenever you click on the Login button in the navbar of  the Feedback page this bug is showing . Please try to solve this issues

**Review by apu52 — CHANGES_REQUESTED** [review]

File: feedback.html (hunk 1 above)
Comment (SadafKausar2025) [reply feedback.html:33]: Ok I will correct it

**Comment by SadafKausar2025** [comment]
Hi @apu52 
I have review it , I saw that the Login page itself has this issue.
when I click login from home page then also getting same error.
for this can I create new issue or solve in this issue only?

**Comment by apu52** [comment]
> Hi @apu52 I have review it , I saw that the Login page itself has this issue. when I click login from home page then also getting same error. for this can I create new issue or solve in this issue only?

I have also rechecked it . there is some problem in this PR . SO , I would like to suggest you to make a complete new PR by closing this PR & by making all the necessary changes

**Comment by SadafKausar2025** [comment]
hello @apu52 
I have resolved the conflict here itself
please merge this pr

I will do the work on another issue aswell

**Comment by SadafKausar2025** [comment]
for login page I will do it in another PR 
because that will be another issue
Kindly please merge this PR

**Comment by apu52** [comment]
> hello @apu52 I have resolved the conflict here itself please merge this pr
> 
> I will do the work on another issue aswell

I can see now there is also a problem in the footer portion , testimonial section now in this PR. So I am closing this PR now . 
Make a new PR by making a new branch in your forked repo & push all the changes there.

**Review by apu52 — COMMENTED** [review]
.

**Pull request closed without merging**